src/tree/param.h

```diff
@@ -76,6 +76,8 @@ struct TrainParam : public dmlc::Parameter<TrainParam> {
   int gpu_id;
   // number of GPUs to use
   int n_gpus;
+  // enable single process per GPU in NCCL
+  bool distributed;
```

Comment:
This was previously `distributed_dask`. Serves the same purpose. Will be removed in a future commit.